### PR TITLE
188 transaction list icons

### PR DESCRIPTION
**Pull request by sirpy**
# Description
- changed streams in collective tx list to fixed amount per month
- added icons per tx type
- added explorer links to superfluid for streams

### CODE REVIEW

File: packages/app/src/components/TransactionList/SupportTransactionListItem.tsx [hunk 1]
```diff
@@ -24,30 +32,35 @@ export function SupportTransactionListItem({ transaction }: SupportTransactionLi
...
   const flowingAmount = useMemo(() => {
     return transaction.isFlowUpdate ? (
-      <FlowingBalance
-        balance={transaction.contribution}
-        balanceTimestamp={timestamp}
-        flowRate={transaction.flowRate}
-        tokenPrice={undefined}
-        style={styles.amount}
-      />
+      <Text style={styles.amount}>
```
Comment (L03TJ3) [inline packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:35]: This should be its own isolated component and used in TransactionListItem, not passed down.

You want to memoize the calculation / update of the text component.

Now it re-renders the whole tree

**Review by L03TJ3 — CHANGES_REQUESTED** [review]

File: packages/app/src/components/TransactionList/SupportTransactionListItem.tsx (hunk 1 above)
Comment (sirpy) [reply packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:35]: can you explain what is the difference between passing down the component vs just the calculation?

Comment (L03TJ3) [reply packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:35]: Well, basically it does not matter if its the component, or just the calculation (my sentence might read or have been written wrong)

you don't want to memoize it here but further down.

That way you isolate the updating/re-rerender to just that part of the component.

added benefit is abstracting the calculation to be used else-where (when needed).

Comment (L03TJ3) [reply packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:35]: In example
```
// Rerenders everything
const TestComponent = () => {
	const SomeMemo = useMemo(() => const calculation = 1x1 = 2; return <Text>{calculation}</Text>, [deps])
	return (
		<div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
			</div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
			</div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
				<SomeMemo />
			</div>
		</div>
	)
}

//Re-renders just SomeMemo
const SomeMemo = () => useMemo(() => const calculation = 1x1 = 2; return <Text>{calculation}</Text>, [deps])

const TestComponent = () => {
	return (
		<div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
			</div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
			</div>
			<div>
			
				<Text>Some text</Text>
				<Icon>SomeIcon</Text>
				<SomeMemo />
			</div>
		</div>
	)
}
```

Comment (sirpy) [reply packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:35]: I dont think you are right. we can discuss it on our call.

**Pull request merged**